client/client-lib/src/ln/db.rs

```diff
@@ -104,3 +106,19 @@ impl_db_lookup!(
     key = LightningGatewayKey,
     query_prefix = LightningGatewayKeyPrefix
 );
+
+#[derive(Debug, Encodable, Decodable, Serialize)]
+pub struct OutgoingPaidInvoiceKey(pub Sha256Hash);
```

Comment:
Maybe call it something like `OutgoingContractPendingKeyPrefix`? Because at this point the lightning invoice isn't paid yet because the gateway could still fail to pay it ...

Reply:
Sure, I'll rename that. Should I change `Sha256Hash` to `ContractId` in that case?